ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -19,61 +19,71 @@
+                <dd class="cell-min-height"> @Model.DateOfNotification </dd>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <dt class="bold-label"> TB service </dt>
-                <dd> @Model.Notification.TBServiceName </dd>
+                <dd> @Model.TBService </dd>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <dt class="bold-label"> Case manager </dt>
-                <dd class="cell-min-height"> @Model.Notification.Episode.CaseManager </dd>
+                <dd class="cell-min-height"> @Model.CaseManager </dd>
             </nhs-grid-column>
         </nhs-grid-row>
     </div>
     <div class="notification-banner-body">
         <nhs-grid-row classes="notification-overview-details">
             <nhs-grid-column grid-column-width="OneQuarter" classes="notification-banner-field">
                 <div class="bold-label"> Name </div>
-                <div class="cell-min-height"> @Model.Notification.FullName </div>
+                <div class="cell-min-height"> @Model.Name </div>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <div class="bold-label"> Date of birth </div>
-                <div class="cell-min-height"> @Model.Notification.FormattedDob </div>
+                <div class="cell-min-height"> @Model.DateOfBirth </div>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <div class="bold-label"> Country of birth </div>
-                <div class="cell-min-height"> @Model.Notification.CountryName </div>
+                <div class="cell-min-height"> @Model.CountryOfBirth </div>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <div class="bold-label"> Postcode </div>
-                <div> @Model.Notification.FormattedNoAbodeOrPostcodeString </div>
+                <div> @Model.Postcode </div>
             </nhs-grid-column>
         </nhs-grid-row>
         <nhs-grid-row classes="notification-overview-details">
             <nhs-grid-column grid-column-width="OneQuarter">
                 <dt class="bold-label"> NHS number </dt>
-                <dd class="cell-min-height">  @Model.Notification.FormattedNhsNumber </dd>
+                <dd class="cell-min-height">  @Model.NHSNumber </dd>
             </nhs-grid-column>
             <nhs-grid-column grid-column-width="OneQuarter">
                 <div class="bold-label"> Sex </div>
-                <div class="cell-min-height"> @Model.Notification.SexLabel </div>
+                <div class="cell-min-height"> @Model.Sex </div>
             </nhs-grid-column>
         </nhs-grid-row> 
     </div>
-    <div class="notification-banner-triangle"> </div>
+    @if(!@Model.SearchPage) 

```

Comment:
Let's move these bits outside the banner - they don't depend on it and are actually just part of the notification page